docs/docs/concepts/advanced/data_structures/indexed_merkle_tree.md

```diff
@@ -9,6 +9,10 @@ import Disclaimer from "../../../misc/common/\_disclaimer.mdx";
 
 ## Indexed Merkle Trees
 
+This article will introduce the concept of an indexed merkle tree, and how it can be used to improve the performance of nullifier trees in circuits. The content was also covered in a presentation with the [Privacy + Scaling Explorations](https://pse.dev/).
```

Comment:
```suggestion
This article will introduce the concept of an indexed merkle tree, and how it can be used to improve the performance of nullifier trees in circuits. The content was also covered in a presentation for the [Privacy + Scaling Explorations team at the Ethereum Foundation](https://pse.dev/).
```